src/topp/MSGFPlusAdapter.cpp

```diff
@@ -208,6 +219,8 @@ class MSGFPlusAdapter :
 
     registerFlag_("legacy_conversion", "Use the indirect conversion of MS-GF+ results to idXML via export to TSV. Try this only if the default conversion takes too long or uses too much memory.", true);
 
+    registerInputFile_("conf", "<file>", "", "Optional MSGF+ configuration file (passed as -conf <file> to MSGF+). See documentation for examples.", false, false);
```

Comment:
do we need to document here how deviating options set on command line and in config are handled?

Reply:
I briefly looked at the msgf docs but no info.
We'd need to test that (hoping they do not change the behavior)

Reply:
one way would be to allow either conf or the other params